src/controllers/words.js

```diff
@@ -43,6 +43,11 @@ export const searchWordUsingEnglish = async ({ query, searchWord, ...rest }) =>
   return sortDocsBy(searchWord, words, 'definitions[0]');
 };
 
+function getWordSearchFunction(searchWord) {
```

Comment:
We want to create functions like so:
```
const getWordSearchFunction = (searchWord) => {}
```
instead of using the `function` keyword.